### DISCUSSION TITLE
Having issues decrypting data backed up from `gocrypt -reverse`

### DISCUSSION THREAD

**Discussion by DurvalMenezes** [rfjakob/gocryptfs#842] [Q&A] [answered]
As per the subject. I think it may be a bug, so I opened an issue here: https://github.com/rfjakob/gocryptfs/issues/841

Has anyone ever seem anything similar? Am I doing something wrong? Any tips? 

TIA!

**Comment by rfjakob** [comment, marked as answer]
Solved in https://github.com/rfjakob/gocryptfs/issues/841

**Reply by rfjakob** [reply]
...solved via documentation update here https://github.com/rfjakob/gocryptfs/commit/7883d383d842707994b664f4d56ffbe833d3dc52
